opensearch_dashboards.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "observabilityDashboards",
   "version": "2.11.0.0",
-  "opensearchDashboardsVersion": "2.11.0",
+  "opensearchDashboardsVersion": "opensearchDashboards",
```

Comment:
This change is causing build to fail. You may want to revert this or update the github workflow.

Reply:
Yeah, mistaken commit.  Will re-post.